test/unit/package_set_test.rb

```diff
@@ -32,6 +32,28 @@ class PackageSetTest < Minitest::Test
       assert_nil(@package_set.fetch("components/unknown"))
     end
 
+    test ".package_paths supports a path wildcard" do
+      package_paths = PackageSet.package_paths("test/fixtures/skeleton", "**")
+      assert_includes(package_paths, Pathname.new("test/fixtures/skeleton/components/sales/package.yml"))
+      assert_includes(package_paths, Pathname.new("test/fixtures/skeleton/package.yml"))
+    end
+
+    test ".package_paths supports a single path as a string" do
+      package_paths = PackageSet.package_paths("test/fixtures/skeleton", "components/sales")
+      assert_equal(package_paths, [Pathname.new("test/fixtures/skeleton/components/sales/package.yml")])
+    end
+
+    test ".package_paths supports many paths as an array" do
+      package_paths = PackageSet.package_paths("test/fixtures/skeleton", ["components/sales", "."])
+      assert_equal(
+        package_paths,
+        [
+          Pathname.new("test/fixtures/skeleton/components/sales/package.yml"),
+          Pathname.new("test/fixtures/skeleton/package.yml"),
+        ]
+      )
```

Comment:
I think it would be good to seperate the setup-action-assertion with whitespace to make the tests easier to read and to make diffing easier.

This approach to testing is also recommended by the ruby shopify style guide:
> Organize the setup, action, and assertion sections of the test case into paragraphs separated by empty lines.

https://github.com/Shopify/ruby-style-guide#testing